R/pkg/R/generics.R

```diff
@@ -623,6 +623,10 @@ setGeneric("getItem", function(x, ...) { standardGeneric("getItem") })
 
 #' @rdname column
 #' @export
+setGeneric("isNaN", function(x) { standardGeneric("isNaN") })
```

Comment:
@felixcheung I agree to not expose ```isnull``` and ```isnotnull```. Could you tell me how to update the release note to show ```isNaN``` is a breaking change?